### ISSUE TITLE
Switching to console/editor with ctrl+shift+c (ctrl+shft+e) does not work when undocked

### ISSUE THREAD

**Issue by martinwk** [spyder-ide/spyder#3292]
When undock the console from the main window in spyder I can not jump from editor to console using the hot keys ctrl+shif+e (ctrl+shift+c). If I close the undocked console, I can open it again with ctr+shft+c, but it does not become the active window. It does work when both are docked though.

**Comment by ccordoba12** [comment]
I don't think we could make this work because when panes are undocked they are separated from the main application, and so we can't use shortcuts to focus them.

@goanpeca, what do you think?

**Comment by martinwk** [comment]
I see, then I should just get a bigger screen to fit everything in :) 

Some less ideal solution would be that all the undocked panes show up in the windows task bar as separate windows, just like an opened e-mail is a separate window of your e-mail client (but all belonging to the same program) I can easily scroll to these windows of the same program with the key combination WND-#. Here WND is the windows key and # the position of the program in the taskbar.

**Comment by goanpeca** [comment]
>I don't think we could make this work because when panes are undocked they are separated from the main application, and so we can't use shortcuts to focus them.

Yep, I think that is completely normal to expect.

**Comment by goanpeca** [comment]
>Some less ideal solution would be that all the undocked panes show up in the windows task bar as separate windows

Yes, there was work I started a long time ago, but the approach I followed was [clunky](https://github.com/spyder-ide/spyder/pull/2288), so I restarted from scratch, so it wont be ready for 3.0, but it will come :-)
